app/graphql/types/payment_providers/adyen.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Types
+  module PaymentProviders
+    class Adyen < Types::BaseObject
+      graphql_name 'AdyenProvider'
+
+      field :id, ID, null: false
+      field :api_key, String, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `api_key` should appear before `id`.